### ISSUE TITLE
 Web-part is not loading for most users returning an error Microsoft 365 learning pathways has a configuration issue. Ask your administrator for assistance

### ISSUE THREAD

**Issue by OlgaKislitsyna** [pnp/custom-learning-office-365#385]
### Describe the bug

We installed Microsoft 365 learning pathways to our tenant using SharePoint look book provisioning. And everything looked fine, but after we gave access to other users (not just the IT Team), it appeared that for the users in one of our offices the web-part is not loading returning an error "Microsoft 365 learning pathways has a configuration issue. Ask your administrator for assistance". 
I attached the messages from console filtered with errors only and the messages with the default filtering. 
Also I attached a screenshot from the network tab. I thought it is also not normal that I didn't see anything when filtering for metadata.json
I suspect this problem could be due to some policies installed on the corporate laptops, as the problem is localised only to one of our offices. But can you help maybe you can point us what policies may cause these errors. Thank you. 

### Learning Pathways version number

4.1.3.0

### Screenshots and console messages

[xxx.sharepoint.com-1603360622476-errors only.log](https://github.com/pnp/custom-learning-office-365/files/5428748/xxx.sharepoint.com-1603360622476-errors.only.log)

[xxx.sharepoint.com-1603360661193 (DefaultLevels).log](https://github.com/pnp/custom-learning-office-365/files/5428747/xxx.sharepoint.com-1603360661193.DefaultLevels.log)

[MetadataFilter_Screenshot.docx](https://github.com/pnp/custom-learning-office-365/files/5428750/MetadataFilter_Screenshot.docx)

**Comment by junterpfe** [comment]
One of my customers is having the same issue
![training error](https://user-images.githubusercontent.com/47872678/97006366-c2afee80-150d-11eb-918a-7d2462391e70.png)

**Comment by juliemturner** [comment]
@OlgaKislitsyna - Based on your errors only log yes, it's showing that there is some sort of policy blocking access to GitHub:

>CustomLearningAdmin.aspx:1 Access to fetch at 'https://pnp.github.io/custom-learning-office-365/learningpathways/v4/manifest.json' from origin 'https://crhemea.sharepoint.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
custom-learning-admin-web-part_7b54fdb5adf76a68373a127035fc2de3.js:1 Message: TypeError: Failed to fetch - InitService (getManifest)

That said, I'm not sure why they are trying to go to the CustomLearningAdmin.aspx page?  Are people in that office meant to be admins?  If so then you need to correct the network policy.

@junterpfe -- Unfortunately the reason you get that error is unique to every situation as there are a lot of different reasons for it. If you follow the [instructions here](https://github.com/pnp/custom-learning-office-365/blob/master/webpart/DiagnosisPreCheck.md) and submit a new issue I can better assist you.

**Comment by NicolasRoelofs** [comment]
Same issue cf. in attachment
v 4.1.3.0
![Capture_Metadata_Filter_404](https://user-images.githubusercontent.com/57498304/98019451-0c4be380-1e02-11eb-8ead-d672dd02d3b7.png)

**Comment by juliemturner** [comment]
@NicolasRoelofs -- Unfortunately the reason you get that error is unique to every situation as there are a lot of different reasons for it. If you follow the [instructions here](https://github.com/pnp/custom-learning-office-365/blob/master/webpart/DiagnosisPreCheck.md) and submit a new issue I can better assist you.

**Comment by NicolasRoelofs** [comment]
> @NicolasRoelofs -- Unfortunately the reason you get that error is unique to every situation as there are a lot of different reasons for it. If you follow the [instructions here](https://github.com/pnp/custom-learning-office-365/blob/master/webpart/DiagnosisPreCheck.md) and submit a new issue I can better assist you.

i'm following the Diagnosis Pre Check and I think I have found, the path of the CDN is incorrect : it should be https://pnp.github.io/sp-custom-learning/learningpathways/v4/ instead of https://pnp.github.io/custom-learning-office-365/learningpathways/v4/ is it?
Thank you

**Comment by NicolasRoelofs** [comment]
i can't modify it
![Capture_Custom_Learning _Properties](https://user-images.githubusercontent.com/57498304/98020880-e0c9f880-1e03-11eb-9719-7054039f1770.png)

**Comment by NicolasRoelofs** [comment]
> i can't modify it
> ![Capture_Custom_Learning _Properties](https://user-images.githubusercontent.com/57498304/98020880-e0c9f880-1e03-11eb-9719-7054039f1770.png)

i will check using  this  script : https://github.com/pnp/custom-learning-office-365/blob/master/webpart/UpdateM365lpCDN.ps1

**Comment by NicolasRoelofs** [comment]
I finally found: it was an "orphan" cdn for SharePoint Success Kit I had to recreate it with the reference id which was mentioned in the log as not being able to be loaded
;-)

**Comment by rhajj19** [comment]
Hello Nicolas , how did you manage to change that ? can you provide me with the steps as i have the same problem :)

**Comment by NicolasRoelofs** [comment]
> Hello Nicolas , how did you manage to change that ? can you provide me with the steps as i have the same problem :)

Hello rhajj19, I followed Julie's diagnostic instructions. In the error logs of the Edge console I identified that the loading did not work for a cdn ID. It was the Sharepoint Success Kit that I had deleted (the path staging_ssk had changed) but that he was still looking to load. I recreated a new package with this ID and everything is back to normal
